flict/flictlib/compatibility.py

```diff
@@ -101,7 +101,7 @@ def check_compatibilities(self, licenses, check_all=False):
             licenses_set = set(licenses + supported)
             outer_licenses = list(licenses_set)
         else:
-            outer_licenses = list(licenses)
+            outer_licenses = list(set(licenses))
```

Comment:
this make the list() obsolete - afais this variable is just used in an iterator, so we can just use the set() instead